src/lib/group_map/group_map.ml

```diff
@@ -1,20 +1,26 @@
-(* Points on elliptic curves over finite fields by M. SKALBA
- * https://www.impan.pl/pl/wydawnictwa/czasopisma-i-serie-wydawnicze/acta-arithmetica/all/117/3/82159/points-on-elliptic-curves-over-finite-fields
+(* we have φ(t) : F -> S
+ * and φ1(λ) : S -> V with
+ * V(F) : f(x1)f(x2)f(x3) = x4^2,
+ * (f is y^2 = x^3 + Bx + C)) (note choice of constant names
+ * -- A is coeff of x^2 so A = 0 for us)
  *
```

Comment:
Should we add the references, so the comments are easier to follow along with?
WB19: Riad S. Wahby and Dan Boneh, Fast and simple constant-time hashing to the BLS12-381 elliptic curve. https://eprint.iacr.org/2019/403
SvdW06: Shallue and van de Woestijne, "Construction of rational points on elliptic curves over finite fields." Proc. ANTS 2006. https://works.bepress.com/andrew_shallue/1/download/

Reply:
Yes, great idea will do.